__tests__/perfume.spec.ts

```diff
@@ -4,7 +4,7 @@
 import { C, WN, WP } from '../src/constants';
 import * as log from '../src/log';
 import { metrics, ntbt } from '../src/metrics';
-import Perfume from '../src/perfume';
+import { Perfume } from '../src/perfume';
```

Comment:
couple of these updates, they are leftover from the perfume export changes

Reply:
nice